src/sidebar/components/test/annotation-body-test.js

```diff
@@ -30,4 +40,48 @@ describe('AnnotationBody', () => {
     assert.isTrue(wrapper.exists('MarkdownEditor'));
     assert.isFalse(wrapper.exists('MarkdownView'));
   });
+
+  it('does not render controls to expand/collapse the excerpt if it is not collapsible', () => {
+    const wrapper = createBody();
+
+    // By default, `isCollapsible` is `false` until changed by `Excerpt`,
+    // so the expand/collapse button will not render
+    assert.notOk(wrapper.find('Button').exists());
+  });
+
+  it('renders controls to expand/collapse the excerpt if it is collapsible', () => {
+    const wrapper = createBody();
+    const excerpt = wrapper.find('Excerpt');
+
+    act(() => {
+      // change the `isCollapsible` state to `true` via the `Excerpt`
+      excerpt.props().onCollapsibleChanged(true);
+      // Force a re-render
+      wrapper.setProps({});
```

Comment:
In this and the following test, I had trouble encouraging the element to re-render without using `setProps`. I'm not sure whether this is a code smell, an enzyme mis-use, or benign.